uwb/example/lib/game_widgets/ball.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+/// The [Ball] class that generates ball on the field.
+class Ball extends StatelessWidget {
+  /// The [Ball] constructor.
+  const Ball(
+      {Key? key, required this.x, required this.y, required this.ballKey})
+      : super(key: key);
+
+  /// Variable for x-axis of the ball.
```

Comment:
It would be more informative to write something like this:
```suggestion
  /// The location on the x-axis at which the ball will be painted.
```